### PR TITLE
Add recipe for rapidnj v2.3.2

### PR DESCRIPTION
**Pull request by johnlees**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).

Slightly unusual test as `rapidnj --help` gives a non-zero exit code, which needs to be ignored

### CODE REVIEW

**Comment by johnlees** [comment]
Request for review @bioconda/core 
Thank you!

File: recipes/rapidnj/run_test.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+rapidnj -h 2> test_stderr.txt || true && diff help.e test_stderr.txt
```
Comment (bgruening) [inline recipes/rapidnj/run_test.sh]: can you include this in the meta.yaml

**Comment by johnlees** [comment]
@bgruening Looks like that doesn't work, as the `source_files` happens after the `command`. I've reverted to the previous passing version, but let me know if I've done something wrong

**Comment by johnlees** [comment]
Could be the language and no dependencies?

**Comment by bgruening** [comment]
Yeah, the failure was due to our build-infrastructure. Please put the test back into the meta.yaml file and we see again.

**Comment by johnlees** [comment]
Seems to need the dedicated `run_test.sh` file

**Comment by johnlees** [comment]
@bgruening is this ok to merge now?

**Comment by johnlees** [comment]
@bioconda/core sorry to ping again, I just wanted to check whether I am ok to merge this as I am keen to list as a dependency in another bioconda recipe
Thanks a lot!

**Comment by johnlees** [comment]
Thanks for the help with this @bgruening! 
I don't understand why bd6ffae failed. Locally, this test command worked and gave a zero exit code

**Comment by bgruening** [comment]
In the end it was your compiler. You used the c compiler but not the c++ one. This results in a missing libgcc library. And this was only catched in our testing container :)

**Comment by bgruening** [comment]
Thanks @johnlees!

**Comment by johnlees** [comment]
Ah great, thanks so much!

**Pull request merged**